docs/application/native/guides/connectivity/usb-host.md

```diff
@@ -240,8 +240,8 @@ dlog_print(DLOG_INFO, LOG_TAG, "%d bytes of response received: %s", bytes_transf
 <a name="async_transfer"></a>
 ## Asynchronous Transfer
 
-Asynchronous transfer is a way to let the system handle traffic in the background, without having to wait for a transfer to finish. A program can request an asynchronous transfer and proceed further, then be notified by the system at some point after the request gets handled.
-Asynchronous transfer includes isochronous and control transfers. For more information, including examples, see [Asynchronous IO](../../api/mobile/latest/group__CAPI__USB__HOST__ASYNC__MODULE.html).
+Asynchronous transfer is a way to allow the system to handle the data traffic in the background without having to wait for the transfer to finish. A program can request an asynchronous transfer and proceed with its operations. The program is notified by the system as soon as the request is handled.
```

Comment:
Please change "the transfer" to "a transfer" as mentioned in the previous comment.
For more information, see https://github.com/Samsung/tizen-docs/pull/1004#discussion_r372739931